webapp/actions/global_actions.jsx

```diff
@@ -593,3 +594,15 @@ export function emitPopoverMentionKeyClick(isRHS, mentionKey) {
         mentionKey
     });
 }
+
+let serverVersion = '';
```

Comment:
The refresh will only occur if the serverVersion is already set, hence the statement `if (serverVersion && serverVersion !== newServerVersion) {`. If it's not already set then it will get set and no refresh will happen. Setting it to `Client4.getServerVersion()` initially is possible but is probably racy because if no client requests have been made yet it will still be blank